Rephrasing configuring (#1390)
codenvy/che-docs: src/main/pages/che-7/installation-guide/con_understanding-the-checluster-custom-resource.adoc

[id="understanding-the-checluster-custom-resource_{context}"]
= Understanding the `CheCluster` Custom Resource

A default deployment of {prod-short} consist in the application of a parametrized `CheCluster` Custom Resource by the {prod} Operator.

`CheCluster` Custom Resource:: 

> * It is a YAML document describing the configuration of the overall {prod-short} installation.

Reviewer: Suggest removing subjects in all of these:
```suggestion
* A YAML document describing the configuration of the overall {prod-short} installation.
```

> * It contains sections to configure each component: `auth`, `database`, `server`, `storage`.

Reviewer: ```suggestion
* Contains sections to configure each component: `auth`, `database`, `server`, `storage`.
```

Role of the {prod} Operator::

> * It translates the `CheCluster` Custom Resource into configuration (ConfigMap) usable by each component of the {prod-short} installation.

Reviewer: ```suggestion
* To translate the `CheCluster` Custom Resource into configuration (ConfigMap) usable by each component of the {prod-short} installation.
```

Role of {orch-name}::

> * It applies the configuration (ConfigMap) for each component.

Reviewer: ```suggestion
* To apply the configuration (ConfigMap) for each component.
```

> * It creates the necessary Pods.

Reviewer: ```suggestion
* To create the necessary Pods.
```

> * When it detects a change in the configuration of a component, it restarts the Pods accordingly.

Reviewer: So we don't have two "it"s in one sentence.

```suggestion
* When {orch-name} detects a change in the configuration of a component, it restarts the Pods accordingly.
```
